contracts/base/LendingPool.sol

```diff
@@ -267,6 +273,13 @@ contract LendingPool is Base, LendingPoolInterface {
         @dev It throws a require error if 'transferFrom' invocation fails.
      */
     function tokenTransferFrom(address from, uint256 amount) private {
+        // The first param is the  owner and second the spender. That's why the unit tests were failing. :D
+        // If I could putting a slapping my head emoji here I would lol :S
+        uint256 allowance = lendingToken.allowance(address(this), from);
```

Comment:
I think the "from" should be first because it is the owner. Then the spender, in this case, the contract address(this).

Reply:
Fixed

Reply:
Fixed